qa/rpc-tests/proxy_test.py

```diff
@@ -83,13 +83,13 @@ def setup_nodes(self):
         # Note: proxies are not used to connect to local nodes
         # this is because the proxy to use is based on CService.GetNetwork(), which return NET_UNROUTABLE for localhost
         args = [
-            ['-listen', '-debug=net', '-debug=proxy', '-proxy=%s:%i' % (self.conf1.addr),'-proxyrandomize=1'], 
```

Comment:
Line 72 `Warning: testing without local IPv6 support` is now a print again, not sure if this was intentional.